### PR TITLE
Kc user profile

### PR DESCRIPTION
**Pull request by kev1n80**
Move css classes that listing and user html files have in common into a card.css file.
Create a user html file that makes the user information sticky on large devices in landscape mode.
Landscape/medium to big screens: 
Screenshot 2020-07-23 at 1 41 32 PM

Portrait/small screens:
Screenshot 2020-07-23 at 1 52 37 PM

### CODE REVIEW

File: src/main/webapp/user.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          <div class="user-listing-container">
+            <div id="created-listings">
+              <!-- From search.html -->
+              <section class="listing shadow-box">
+                <div>
+                  <img class="listing-image" src="" alt="Listing preview image.">
+                  <span class="listing-reputation">Reputation: 205 Votes</span>
+                </div>
+                <div class="listing-info-container">
+                  <div class="listing-heading-container">
```
Comment (zrideaux) [inline src/main/webapp/user.html:32]: Be sure to edit the classes on this page to use your updated names.

Comment (kev1n80) [reply src/main/webapp/user.html:32]: This code is from your search.html file and uses your styling. I wanted to reuse the styling and coding that you already made to keep the website styling similar.

Comment (zrideaux) [reply src/main/webapp/user.html:32]: If that's the case, it doesn't look like the search stylesheet is linked in the head, just so you know!

Comment (kev1n80) [reply src/main/webapp/user.html:32]: Hahaha yes sorry, I added this change in the new pr. I should've added that. Thank you for pointing this out!

**Pull request merged**